ajax/libs/react/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react",
-  "filename": "react.min.js",
-  "version": "15.6.1",
+  "filename": "umd/react.production.min.js",
+  "version": "16.0.0-beta.2",
```

Comment:
Will it pick up the latest 16 tree beta changes in directory structure in CI then?